### PR TITLE
catfish: 1.4.10 -> 1.4.11

### PR DESCRIPTION
**Pull request by romildo**
###### Motivation for this change

- Update to version [1.4.11](https://mail.xfce.org/pipermail/xfce-announce/2019-December/000807.html)

- Move to `pkgs/desktops/xfce/applications`. Since release 1.4.6 [Catfish is an Xfce Project](https://bluesabre.org/2018/07/15/catfish-1-4-6-released-now-an-xfce-project).

[Catfish 1.4.11 Released](https://bluesabre.org/2019/12/21/catfish-1-4-11-released/)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by worldofpeace** [comment]
Python3 https://git.xfce.org/apps/catfish/tree/setup.py#n1?

File: pkgs/desktops/xfce/applications/catfish/default.nix [hunk 1]
```diff
@@ -4,14 +4,14 @@
 }:
 
 pythonPackages.buildPythonApplication rec {
+  pname = "catfish";
   majorver = "1.4";
-  minorver = "10";
+  minorver = "11";
```
Comment (worldofpeace) [inline pkgs/desktops/xfce/applications/catfish/default.nix]: can we just use a version attribute?

**Review by worldofpeace — CHANGES_REQUESTED** [review]

File: pkgs/desktops/xfce/applications/catfish/default.nix (hunk 1 above)
Comment (romildo) [reply pkgs/desktops/xfce/applications/catfish/default.nix]: Done.

**Comment by romildo** [comment]
> Python3 https://git.xfce.org/apps/catfish/tree/setup.py#n1?


It works with Python2 for me.

Should `pythonPackages` be changed to `python3Packages`?

**Comment by worldofpeace** [comment]
> > Python3 https://git.xfce.org/apps/catfish/tree/setup.py#n1?
> 
> It works with Python2 for me.
> 
> Should `pythonPackages` be changed to `python3Packages`?

Yes,  python2 depredations and all, we should default to python3Packages.

**Comment by romildo** [comment]
> > > Python3 https://git.xfce.org/apps/catfish/tree/setup.py#n1?
> > 
> > 
> > It works with Python2 for me.
> > Should `pythonPackages` be changed to `python3Packages`?
> 
> Yes, python2 depredations and all, we should default to python3Packages.

Changed to `python3Packages`.

**Review by worldofpeace — APPROVED** [review]
LGTM.

**Pull request merged**